command/host_create.go

```diff
@@ -61,8 +66,70 @@ func (c *createHost) Execute(ctx context.Context, comm client.Communicator,
 			return err
 		}
 	}
+	startTime := time.Now()
+	ids, err := comm.CreateHost(ctx, taskData, *c.CreateHost)
+	if err != nil || c.CreateHost.CloudProvider != apimodels.ProviderDocker || c.CreateHost.Background == false {
+		return err
```

Comment:
should wrap the error, also for the OR case you should probably move to a second field to avoid returning a nill error

Reply:
I do want to return nil for the second case, but I can separate them to make it less ambiguous.